packages/tools/generator-pie-component/src/app/templates/component.__stories__.ts

```diff
@@ -1,9 +1,10 @@
 import { html, TemplateResult } from 'lit';
 import type { StoryObj as Story } from '@storybook/web-components';
-import { Pie<%= componentName %>, <%= componentName %>Props } from '@justeattakeaway/pie-<%= fileName %>';
+import '@justeattakeaway/pie-<%= fileName %>';
```

Comment:
probably worth checking with @raoufswe if this is still needed

Reply:
is there a reason why we changed this? 
the current approach is being used across all story files and it should work as expected unless I'm missing something 👀

Reply:
```
import { Pie<%= componentName %> } from '@justeattakeaway/pie-<%= fileName %>';
import { <%= componentName %>Props } from '@justeattakeaway/pie-<%= fileName %>/src/defs';
```

This is how it's done in `pie-modal.stories.ts` and I believe gives improved IntelliSense for the types. However, I've lost some context since I was asked to pass this branch and ticket to Faye and Kevin. 

Feel free to delete it and if we still see an issue we can raise a separate PR in the future.

Reply:
I will look into that in a separate PR and add them to the generator if needed

Reply:
Thanks! we will remove it from this PR 😄